storage/tests/unit/test_bucket.py

```diff
@@ -607,217 +607,6 @@ def api_request(cls, *args, **kwargs):
         expected_cw = [((), expected_called_kwargs)]
         self.assertEqual(_FakeConnection._called_with, expected_cw)
 
-    def test_create_w_user_project(self):
```

Comment:
Most part of these tests had already been move into `client_test.py`. Some will be moved with this PR